gempython/utils/db/settings.py

```diff
@@ -0,0 +1,17 @@
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "gempython.utils.db.settings.local")
+SECRET_KEY = 'ef7$_)r0$#(x)ovd^m_6_bevsonkaxalvrhds9@ilb_p0o&-xn'
+
+INSTALLED_APPS = [
+    'gempython.utils.ldqm_db',
+    ]
+DATABASES = {
+    'default': {
+        'ENGINE':  'django.db.backends.mysql',
+        'NAME':     'ldqm_tif_qc8_db',
+        'PORT':     3306,
+        'HOST':     'cosmicstandtif',
+        'USER':     'gemdaq',
+        'PASSWORD': 'gemdaq',
+    }
+}
```

Comment:
Last thing, if I look at the diff, this version is the one that changed (as I had mentioned in several other comments) and it never was corrected, explicitly:
```python
        'NAME':     'ldqm_test_db',		
        'PORT':     3306,		
        'HOST':     'gem904daq01',		
        'USER':     'gemdaq',		
        'PASSWORD': 'gemdaq',
```